string ref_frame
string tg_frame
---
float32 tx
float32 ty
float32 tz
float32 rx
float32 ry
float32 rz
float32 rw

